### PR TITLE
add field acceptQueueSize in jetty-https 

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

File: src/assembly/jetty/etc/jetty-https.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
           <Set name="port"><Property name="https.port" default="443" /></Set>
           <Set name="idleTimeout"><Property name="https.timeout" default="30000"/></Set>
           <Set name="soLingerTime"><Property name="https.soLingerTime" default="-1"/></Set>
+          <Set name="acceptQueueSize">500</Set>
```
Comment (flopezag) [inline src/assembly/jetty/etc/jetty-https.xml]: Just some questions. Currently it is unlimited? And Why 500 and not less or more?

Comment (flopezag) [inline src/assembly/jetty/etc/jetty-https.xml]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**